app/src/main/java/com/github/dedis/student20_pop/ui/AttendeeFragment.java

```diff
@@ -45,14 +46,21 @@
     @Override
     public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
 
-        //for testing purposes:
-        lao = new Lao("LAO I just joined", new Date(), new Keys().getPublicKey());
-
         View rootView = inflater.inflate(R.layout.fragment_attendee, container, false);
-
+        PoPApplication app = (PoPApplication)(this.getActivity().getApplication());
+        lao = app.getCurrentLao();
+        //lao will be null in the tests, as Attendee UI is tested "on its own"
+        //so we set a dummy lao with dummy events for the tests to work correctly
+        if (lao == null){
```

Comment:
A better way to test AttendeeUI would be to mock the function getCurrentLao so you can choose what it returns, it would be better than creating a dummy Lao in the code. Maybe add it in a TODO so we can do it if we have time

Reply:
yep good idea, working on it